src/models/gen_embeddings.py

```diff
@@ -1,11 +1,8 @@
 '''
 This script is a work-in-progress and needs further development.
-Right now all of the functions below have successfully run within
-a notebook that is being used in tandem to develop this script.
-In the next branch, I will continue to build this out, namely by
-adding a config file and also by figuring out a more appropriate
-means by which to store embeddings, most likely in a database
-of some sort (and not within the 'data' sub-folder of the directory)
+It is attempting to save the embeddings as numpy files, and Tweet IDs as
```

Comment:
Attempting to, or doing? Sounds scary to read this in software code. If it "attempts", what does it do if it "fails"? If it always works, just say "Saves the embeddings as numpy files".